jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java

```diff
@@ -2,14 +2,12 @@
 
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
-import org.hamcrest.Matchers;
-import org.testng.Assert;
 
 import static com.epam.jdi.light.driver.WebDriverFactory.*;
 import static com.epam.jdi.light.elements.composite.WebPage.*;
 import static com.epam.jdi.tools.JsonUtils.*;
 import static org.hamcrest.MatcherAssert.*;
-import static org.testng.Assert.*;
+import static org.hamcrest.Matchers.*;
```

Comment:
hamcrest (need testng)

Reply:
Was added in master, are we sure we should remove it?